templates/bbb/bigbluebutton.j2

```diff
@@ -2,9 +2,12 @@ server {
         listen 80;
         listen [::]:80;
         server_name {{ bbb_hostname }};
-        return 301 https://$host$request_uri;
-}
 
+        # If a user tries to come through http, redirect them through https
```

Comment:
this server is listing on 80 without ssl ...
So your patch for this file is useless (sorry)

Reply:
@genofire Without this patch there is no redirect to HTTPS or am I missing anything?

Reply:
there is the line: ` return 301 https://$host$request_uri;` which you try to remove, should solve the redirect allready